### PR TITLE
Updated active admin requirements and incremented version

### PR DESCRIPTION
**Pull request by jaredhales**

### CODE REVIEW

File: questionable.gemspec [hunk 1]
```diff
@@ -7,8 +7,8 @@ require "questionable/version"
 Gem::Specification.new do |s|
   s.name        = "questionable_surveys"
   s.version     = Questionable::VERSION
-  s.authors     = ["Nick Urban, Dorian Marié"]
-  s.email       = ["nick@nickurban.com, dorian@bespokepost.com"]
+  s.authors     = ["Nick Urban, Dorian Marie, Jared Hales"]
```
Comment (Dorian) [inline questionable.gemspec]: Why did you remove the é? :)

Comment (jaredhales) [inline questionable.gemspec]: https://github.com/bespokepost/questionable/issues/9 - I don't know why that character would be an issue. Maybe something in his stack doesn't support unicode characters?

Comment (Dorian) [inline questionable.gemspec]: Interesting, most probably some `<= 1.8.7` ruby env, or windows. The bug is in the environment not in this file.
